space-plugins/nuxt-base/server/utils/webhook.ts

```diff
@@ -28,21 +28,22 @@ export const createStoryblokWebhook: CreateStoryblokWebhook = async (
 	const url = `${apiHost}/v1/spaces/${params.spaceId}/webhook_endpoints`;
 
 	try {
-		const result: StoryblokWebhookResponse = await $fetch(url, {
-			headers: {
-				Authorization: `Bearer ${params.accessToken}`,
-			},
-			method: 'POST',
-			body: {
-				name: params.name,
-				description: params.description,
-				endpoint: params.endpoint,
-				secret: params.secret,
-				actions: params.actions,
-				activated: params.activated ?? true,
-				isLegacy: params.isLegacy ?? false,
-			},
-		});
+		const result: StoryblokWebhookResponse =
+			await $fetch<StoryblokWebhookResponse>(url, {
```

Comment:
@Dawntraoz for me the whole block was underlined red before I added this generic.